backend/tests/manual/test-universal-storage.sh

```
@@ -179,12 +179,8 @@ fi
 # Step 6: Get download URL
 echo ""
 echo "6. Getting download URL..."
-DOWNLOAD_STRATEGY=$(curl -s -X POST $API_URL/api/storage/buckets/$BUCKET_NAME/objects/$KEY/download-strategy \
-  -H "Authorization: Bearer $TOKEN" \
-  -H 'Content-Type: application/json' \
-  -d '{
-    "expiresIn": 3600
-  }')
+DOWNLOAD_STRATEGY=$(curl -s $API_URL/api/storage/buckets/$BUCKET_NAME/objects/$KEY/download-strategy \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)